$MirOS: ports/comms/linphone/patches/patch-osipua_src_sdphandler_c,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- osipua/src/sdphandler.c.orig	Thu Mar 11 17:41:38 2004
+++ osipua/src/sdphandler.c	Fri Oct  1 23:02:45 2004
@@ -22,10 +22,11 @@
 #include "utils.h"
 
 #ifdef INET6
-#include <netdb.h>
-#include <string.h>
 #include <sys/types.h>
 #include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#include <string.h>
 
 #ifdef HAVE_IFADDRS_H
 #include <ifaddrs.h>
